SimplenoteUITests/History.swift

```diff
@@ -0,0 +1,45 @@
+import XCTest
+
+class History {
+
+    class func close() {
+        guard app.staticTexts[UID.Button.restoreNote].waitForExistence(timeout: minLoadTimeout) else { return }
+        app.buttons[UID.Button.dismissHistory].tap()
+    }
+
+    class func setSliderPosition(position: CGFloat) {
+        app.sliders.element.adjust(toNormalizedSliderPosition: position)
+    }
+
+    class func restoreNote() {
+        app.buttons[UID.Button.restoreNote].tap()
+    }
+}
+
+class HistoryAssert {
+    class func historyShown() {
+        let closeHistoryButton = app.buttons[UID.Button.dismissHistory]
```

Comment:
Nitpick (and it's only my opinion): having the button named after the string identifier makes the code easier on the eye:

```suggestion
        let dismissHistoryButton = app.buttons[UID.Button.dismissHistory]
```